Assets/BossRoom/Scripts/Client/Game/State/ClientMainMenuState.cs

```diff
@@ -45,14 +45,12 @@ protected override void InitializeScope()
         [Inject]
         async void InjectDependenciesAndInitialize(AuthenticationServiceFacade authServiceFacade, LocalLobbyUser localUser, LocalLobby localLobby)
         {
-            #if UNITY_EDITOR
-            if (string.IsNullOrEmpty(UnityEditor.CloudProjectSettings.projectId))
+            if (string.IsNullOrEmpty(Application.cloudProjectId))
             {
-                PopupManager.ShowPopupPanel("Unity Game Services ProjectID not set up" ,"Please follow the steps outlined in Readme to set up your project id - it is required for the services to function.");
+                PopupManager.ShowPopupPanel("Unity Game Services ProjectID not set up" ,"Click the Readme file in the Assets Folder within the Project window in-editor to follow \"Setting Up Unity Game Services\"");
```

Comment:
```suggestion
                PopupManager.ShowPopupPanel("Unity Game Services ProjectID not set up" ,"Click the Readme file in the Assets Folder within the Project window in-editor to follow \"How to set up Unity Game Services\"");
```
Just directing to the new section you've created.